app.js

```diff
@@ -0,0 +1,56 @@
+const fs = require('fs');
+
+const isParsing = process.env.AUTO_PARSING === 'true' ? true : false;
+
+const pathCustom = 'grokPatternList.json';
+const pathInternal = 'grokPatternListInternal.json';
+const fluentdFile = 'fluentd/etc/grokParse.conf';
```

Comment:
Is this file is part of the fluentd installation / part of the docker / something that a customer is adding?